src/framework/piral-core/src/helpers.tsx

```diff
@@ -138,10 +138,15 @@ export function createPiletOptions({
       if (process.env.DEBUG_PILET !== undefined) {
         // the DEBUG_PILET env should point to an API address used as a proxy
         const piletApi = process.env.DEBUG_PILET;
```

Comment:
Are you suggesting that we _should not_ use `process.env.DEBUG_PILET` at all if we have `window['dbg:piral-emulator'].piletApi`? 

I keep it here because I didn't want to mess with the current implementation.

Reply:
I would argue to make a clean cut here.

Let's remove `process.env.DEBUG_PILET` and replace it by `window['dbg:pilet-api']`. Also let's not have it as an object, but rather as a string. If that string is empty or undefined I'd fall back to `/$pilet-api`.

How does that sound?

Reasons: Still has the fallback in case its used with older Piral CLIs. Does not require (at least for some cases) an environment variable (which is only the mechanism for things known at compile-time; this thing, however, should be flexible afterwards). Follows the naming convention as beforehand, is simple, is predictable.

Reply:
I addressed your feedback in the latest revision. I _think_ it's good but you can correct me here.